app/models/ad.rb

```diff
@@ -50,4 +50,8 @@ def medical_help?
   def to_param
     [id, kind.parameterize, city.parameterize].join('-')
   end
+
+  def maps_query
+    [address.gsub(' ', '+'),zip, city].join(',+')
```

Comment:
This should avoid extra `,` if something is blank.

```suggestion
    [address.gsub(' ', '+'), zip, city].select(:present?).join(',+')
```